package.json

```diff
@@ -1,80 +1,55 @@
 {
-	"name": "listr",
-	"version": "0.14.3",
+	"private": true,
+	"version": "0.15.0",
 	"description": "Terminal task list",
 	"license": "MIT",
 	"repository": "SamVerschueren/listr",
 	"author": {
 		"name": "Sam Verschueren",
 		"email": "sam.verschueren@gmail.com",
-		"url": "github.com/SamVerschueren"
+		"url": "https://github.com/SamVerschueren"
 	},
-	"engines": {
-		"node": ">=10"
-	},
-	"scripts": {
-		"test": "clinton && xo && nyc ava",
-		"lint:staged": "lint-staged"
-	},
-	"files": [
-		"index.js",
-		"lib"
+	"workspaces": [
+		"packages/*"
 	],
-	"keywords": [
-		"cli",
-		"task",
-		"list",
-		"tasklist",
-		"terminal",
-		"term",
-		"console",
-		"ascii",
-		"unicode",
-		"loading",
-		"indicator",
-		"progress",
-		"busy",
-		"wait",
-		"idle"
-	],
-	"dependencies": {
-		"@samverschueren/stream-to-observable": "^0.3.0",
-		"is-observable": "^2.0.0",
-		"is-promise": "^2.1.0",
-		"is-stream": "^1.1.0",
-		"listr-silent-renderer": "^1.1.1",
-		"listr-update-renderer": "^0.5.0",
-		"listr-verbose-renderer": "^0.6.0",
-		"p-map": "^2.0.0",
-		"rxjs": "^6.3.3"
+	"scripts": {
+		"build": "tsc -b",
+		"build:watch": "tsc -b -w",
+		"lint": "eslint . --ext=js,ts",
+		"test": "jest",
+		"test:ci": "yarn lint && yarn test",
+    "lint:staged": "lint-staged",
+    "postinstall": "yarn build"
 	},
 	"devDependencies": {
-		"ava": "*",
-		"clinton": "*",
-		"codecov": "^3.1.0",
+		"@babel/core": "^7.9.6",
+		"@babel/preset-env": "^7.9.6",
+		"@babel/preset-typescript": "^7.9.0",
+		"@types/jest": "^25.2.1",
+		"@types/split": "^1.0.0",
+		"@types/zen-observable": "^0.8.0",
+		"@typescript-eslint/eslint-plugin": "^2.30.0",
+		"@typescript-eslint/parser": "^2.30.0",
+		"babel-jest": "^25.5.1",
 		"delay": "^4.1.0",
-		"hook-std": "^1.1.0",
-		"lint-staged": "^8.0.5",
+		"eslint": "^6.8.0",
+		"eslint-config-prettier": "^6.11.0",
+		"eslint-plugin-import": "^2.20.2",
+		"eslint-plugin-prettier": "^3.1.3",
+		"jest": "^25.5.4",
+		"lerna": "^3.20.2",
+		"lint-staged": "^10.2.2",
 		"log-symbols": "^2.2.0",
-		"nyc": "^13.1.0",
 		"pre-commit": "^1.2.2",
+		"prettier": "^2.0.5",
+		"rxjs": "6.x",
 		"split": "^1.0.1",
-		"xo": "*",
```

Comment:
Do not change our code style.

Reply:
It was a way of updating the linter to fit the current standards and ecosystem. XO has 30k downloads vs eslint/prettier who have more than two million.  A customized eslint setup with prettier and @typescript-eslint is much more powerful and flexible than xo. Let's use whatever coding styles you like, but don't use outdated and unused tools for linting when you have a clear industry standard (eslint) that provides by far the best DX, features, and IDE integrations.

Reply:
Sorry I can't agree with you. XO is based on ESLint. It is inappropriate for you to try to compare them. We can continue to use XO's code style, and it also supports TypeScript.

Please do not remove AVA and XO, we do not want to change in this regard. Sam and I use AVA and XO for all our open-source work and we like them.

Reply:
Ok, feel free to change it. Just so you know I doubt XO has as good  typescript rules as typescript-eslint. And AVA sure lacks a lot of features jest has to support this type of typescript monorepo. I wouldn't stick with them for too long if I was you.